lib/cocina/models/builders/title_builder.rb

```diff
@@ -27,92 +27,127 @@ def self.build(titles, strategy: :first, add_punctuation: true)
         #   we can boost matches on it in search results (boost matching this string higher than matching full title string)
         #   e.g. "The Hobbit" (main_title) vs "The Hobbit, or, There and Back Again (full_title)
         # @param [[Array<Cocina::Models::Title,Cocina::Models::DescriptiveValue>] titles the titles to consider
-        # @return [String] the main title value for Solr
+        # @return [Array<String>] the main title value(s) for Solr - array due to possible parallelValue
         def self.main_title(titles)
           new(strategy: :first, add_punctuation: false).main_title(titles)
         end
 
         # the "full title" is the title WITH subtitle, part name, etc.  We want to able able to index it separately so
         #   we can boost matches on it in search results (boost matching this string higher than other titles present)
         # @param [[Array<Cocina::Models::Title,Cocina::Models::DescriptiveValue>] titles the titles to consider
-        # @return [String] the title value for Solr
+        # @return [Array<String>] the full title value(s) for Solr - array due to possible parallelValue
         def self.full_title(titles)
-          new(strategy: :first, add_punctuation: false).build(titles)
+          [new(strategy: :first, add_punctuation: false, only_one_parallel_value: false).build(titles)].flatten.compact
         end
 
         # "additional titles" are all title data except for full_title.  We want to able able to index it separately so
         #   we can boost matches on it in search results (boost matching these strings lower than other titles present)
         # @param [[Array<Cocina::Models::Title,Cocina::Models::DescriptiveValue>] titles the titles to consider
         # @return [Array<String>] the values for Solr
         def self.additional_titles(titles)
-          new(strategy: :all, add_punctuation: false).build(titles) - [full_title(titles)]
+          [new(strategy: :all, add_punctuation: false).build(titles)].flatten - full_title(titles)
         end
 
-        def initialize(strategy:, add_punctuation:)
+        def initialize(strategy:, add_punctuation:, only_one_parallel_value: true)
           @strategy = strategy
           @add_punctuation = add_punctuation
+          @only_one_parallel_value = only_one_parallel_value
```

Comment:
only_one_parallel_value is to keep the behavior of the "build" method the same ... to keep the formerly existing methods behaving the same.   I borked this when I released 0.93.1 ... my bad.

Reply:
documented now